package.json

```diff
@@ -49,15 +46,12 @@
     "mobx-react": "6.3.0",
     "prettier": "^3.0.0",
     "react": "^17.0.1",
-    "react-addons-css-transition-group": "15.6.2",
     "react-autobind": "^1.0.6",
     "react-dom": "^17.0.1",
     "react-json-tree": "0.13.0",
     "react-keydown": "1.9.12",
-    "react-linkify": "1.0.0-alpha",
-    "react-virtualized": "^9.22.2",
+    "react-virtualized": "^9.22.4",
     "tmp": "0.2.1",
-    "tslib": "2.0.3",
```

Comment:
This was used with the `importHelpers` flag in `tsconfig.json` but that seems to conflict with the `noEmitHelpers` flag being on.

> No Emit Helpers - noEmitHelpers
> Instead of importing helpers with [importHelpers](https://www.typescriptlang.org/tsconfig/#importHelpers), you can provide implementations in the global scope for the helpers you use and completely turn off emitting of helper functions.

https://www.typescriptlang.org/tsconfig/#noEmitHelpers